backend/core/detection.py

```diff
@@ -219,6 +218,8 @@ def __init__(self, connection: Connection) -> NoReturn:
 
             self.rtrmanager = None
             if RPKI_VALIDATOR_ENABLED == "true":
+                from rtrlib import RTRManager
```

Comment:
isn't this already imported in another part of the code?

Reply:
removed and made it optional